api/test/network-info-logger.test.js

```diff
@@ -0,0 +1,59 @@
+import assert from 'node:assert'
+import { Point } from '../../common/telemetry.js'
+import { logNetworkInfo } from '../lib/network-info-logger.js'
+
+const telemetry = []
+const recordTelemetry = (measurementName, fn) => {
+  const point = new Point(measurementName)
+  fn(point)
+  telemetry.push(point)
+}
```

Comment:
Please use the test helper I introduced in #354 and made available to API tests in #355.

Also, the list of recorded telemetry points should be reset between the tests, so it's best to move the initialisation into each test.

```js
it('should record new network info if not present for the day', async () => {
  const {telemetry, recordTelemetry} = createTelemetryRecorderStub()
  await logNetworkInfo(headers1, 'station-id1', recordTelemetry)
  // etc.
})